firmware/application/apps/ert_app.cpp

```diff
@@ -38,9 +38,10 @@ namespace format {
 std::string type(Packet::Type value) {
 	switch(value) {
 	default:
-	case Packet::Type::Unknown:	return "???";
-	case Packet::Type::IDM:		return "IDM";
-	case Packet::Type::SCM:		return "SCM";
+	case Packet::Type::Unknown:	return "??? ";
```

Comment:
What's the reason for the spaces on the end?

Reply:
This string function is currently unused as the protocol is not currently being printed in RecentEntriesTable<ERTRecentEntries>::draw, however if someone in the future decided to add the protocol to the print string (see line 86), ideally all of the protocol description strings should be the same number of characters.

Reply:
I've added another minor change to this file below to print the protocol type to the ert.txt log file (similar to the tpms.txt log file except the much shorter description strings), making use of the function mentioned above.  If you prefer the extra space padding be removed from the strings, please let me know.

Reply:
Looks ok to me. Maybe you can add a comment in the code for the extra space ?
For me that would be enough.

Reply:
I just updated the file to remove them since they're not needed in the current display screen.